### DISCUSSION TITLE
NYC Shooting Incidents

### DISCUSSION THREAD

**Discussion by Zhiyuan-Jiao** [jakobzhao/geog495#68] [Final Project]
Link: https://zhiyuan-jiao.github.io/NYC-Shooting-Incidents/
Github: https://github.com/Zhiyuan-Jiao/NYC-Shooting-Incidents

This is a breakdown of every shooting incident that occurred in NYC going back to 2006 through the end of the previous calendar year. This data is manually extracted and reviewed by the Office of Management Analysis and Planning before being posted on the NYPD website. One of the main function and purpose of this map is to get a better understanding of the shooting incidents that has already happen around them in the neighborhood by put down their own addresses. Thus, more people could address what we know about gun violence around them and prevents it from occurring in the first place.

<img width="1440" alt="Screen Shot 2021-12-12 at 9 23 24 PM" src="https://user-images.githubusercontent.com/56005749/145757029-13a487ea-6663-40d7-b81c-0b24d3082901.png">

**Comment by LiamGearyUW** [comment]
Quite a good map, and using dots to represent shootings makes sense. When zooming out, one can see larger trends.

I'm not sure if this is possible with what we've learned in this class, but if you could, I'd recommend making it so that multiple dots in one place become a larger dot with a different color. As is, it's hard to differentiate a single shooting in one place from several shootings in another, and I likely wouldn't have noticed any difference had I not zoomed in and seen slight differences in opacity.

**Comment by nlertv** [comment]
I like how you incorporate geocoder into your project. It is really helpful to be able to search the incidents close to a specific location. Also, I like how you include where you get your sources from as well. The webpage also looks very clean and easy to look at. To improve, I think it might be helpful to include information on each of the incidents. It can be a pop-up that tells you the date of the incident. I think this will be really helpful for the user.

**Comment by seanhy1** [comment]
I really enjoyed interacting with your map to discern the various visual trends you have created utilizing the given data in your web application. To provide more intuitive and accessible information about the plotted data points is to figure out a to provide any information in regards to each shooting incident. For example, if I were to click/hover over a data point, it may display the year of which the shooting occurred as a text label somewhere under your title in your application's user interface. This would provide more interactivity and engage the audience to properly discern some sort of story/narrative for each plot you have displayed as of right now. Besides this however, very well done!

**Comment by yodapancake** [comment]
I enjoyed looking at your map and seeing where shootings over the past 15 years in New York have happened. One thing that could be improved is the base map. In the screenshot of your map the buildings are extruded in 3d which I really enjoy and think adds creative value to the map. Another that could be improved is adding functionality with a time slider to visualize the data over time(i.e. see where shootings have occurred over the years).

**Comment by Conibaer** [comment]
This is a great map with a very clear portrayal of data. I appreciate that information about the data itself is included on the map and in your description. It's interesting to see all shooting incidents that didn't result in death. Coloring the points differently or even adding a toggle option to delineate between fatal and non fatal shootings would be an interesting feature.

**Comment by shimizusUW** [comment]
Some ideas for improvements could include having a feature where there can be filters for different types of incidents. Furthermore, possibly providing more information about each incident with the pointer once clicked will allow for better use of this application. Overall, I love the design on the map!

**Comment by RensenJiang** [comment]
I really like your map, it helps people identify the dangerous area. Some improvements could be adding a filter function and using different map icons to differentiate the area's shooting rate.
